freedom-makeattributes-generator.c++

```diff
@@ -249,17 +249,31 @@ static void write_config_file (fstream &os, std::string board, std::string relea
 
     isa = arch2arch(get_dts_attribute("/cpus/cpu@0", "riscv,isa"));
     std::size_t found = isa.find("rv64");
+
+    // Get the CPU compatible string
+    cpucompat = get_dts_attribute("/cpus/cpu@0", "compatible");
```

Comment:
Rather than fix on cpu0, we should base on the chosen node specified in the DTS file.
See Nate, latest merge

Reply:
+1 on this

Reply:
Get the chosen node from DTS.